apps/examples/wifi_manager_sample/wm_test_stress_test1.c

```diff
@@ -122,20 +121,20 @@ static void wm_get_apinfo(wifi_manager_ap_config_s *apconfig)
 TEST_SETUP(init)
 {
 	ST_START_TEST;
-	ST_EXPECT_EQ(WIFI_MANAGER_SUCCESS, wifi_manager_init(&g_wifi_callbacks));
 	ST_END_TEST;
 }
 
 TEST_TEARDOWN(init)
 {
 	ST_START_TEST;
+	ST_EXPECT_EQ(WIFI_MANAGER_SUCCESS, wifi_manager_deinit());
 	ST_END_TEST;
 }
 
 TEST_F(init)
 {
 	ST_START_TEST;
-	ST_EXPECT_EQ(WIFI_MANAGER_SUCCESS, wifi_manager_deinit());
+	ST_EXPECT_EQ(WIFI_MANAGER_SUCCESS, wifi_manager_init(&g_wifi_callbacks));
```

Comment:
the purpose of TEST(init) is check not only deinit result but also time duration of deinit. So deinit must be in TEST_F(init) function.